build.gradle

```diff
@@ -11,12 +14,13 @@ dependencies {
   compile 'com.github.AbletonDevTools:jenkins-pipeline-mocks:0.7+'
   compile 'org.codehaus.groovy:groovy-all:2.4.12'
 
+  testCompile 'org.assertj:assertj-core:3.15+'
```

Comment:
Same here, I don't get how this is related to updating the version number.

Reply:
TBH I also don't completely understand why this is necessary, but it seems to be related to how dependencies are managed by JenkinsPipelineUnit. I've filed an issue about that here: https://github.com/jenkinsci/JenkinsPipelineUnit/issues/209.

Regardless, this issue would be fixed in 1.5, so to bump to 1.4 we'd still need the `assertj-core` dependency.

Reply:
Understood—it would be worth adding a comment explaining and linking to the issue you filed, I think.